silhouette/test/com/mohiva/play/silhouette/impl/authenticators/JWTAuthenticatorSpec.scala

```diff
@@ -63,7 +63,7 @@ class JWTAuthenticatorSpec extends PlaySpecification with Mockito with JsonMatch
       val jwt = service(None).serialize(authenticator)
       val json = Base64.decode(jwt.split('.').apply(1))
 
-      json must /("exp" -> authenticator.expirationDate.getMillis / 1000)
+      json must /("exp" -> (authenticator.expirationDate.getMillis / 1000).toInt) //FIXME There is not a implicit from Long to JsonSelect in Specs2
```

Comment:
Spec2 3.4 there isn't a implicit to convert `Long` to `JsonSelector`

https://github.com/etorreborre/specs2/blob/master/matcher-extra/src/main/scala/org/specs2/matcher/JsonMatchers.scala#L339-L344